package.json

```diff
@@ -16,7 +16,8 @@
   },
   "homepage": "https://github.com/doesangueorg/doesangueorg.github.io#readme",
   "dependencies": {
-    "gulp": "^3.9.1"
+    "gulp": "^3.9.1",
+    "gulp-install": "^1.1.0"
```

Comment:
I cannot understand where we use the `gulp-install` package. It's really needed?